appium/README.md

```diff
@@ -10,12 +10,16 @@
 6. `nvm install 16` - installs NodeJS 16 and sets it as default
 7. `cd ~/git/flowcrypt-ios/appium && npm install`
 
-## Run tests
+## Building app for testing
+
+Run this in `flowcrypt-ios` folder: `bundle exec fastlane build`. This will produce folder `appium/FlowCrypt.app` that contains the built app.
 
-Run this in `appium` folder. 
+## Run tests
 
-`npm test` - build `FlowCrypt.app` and run all ui tests
+Run these in `appium` folder. `live` means real production APIs, `mock` means local mock APIs. 
 
-`npm run-script only.test.all` - run all ui tests without building the `.app`. Use this if you already built the `.app` before, and now only want to change the UI test spec without rebuilding the app
+To run a particular test:
+- `npm run-script test.live "user is able to view text email"`
+- `npm run-script test.mock "app setup fails with bad EKM URL"`
```

Comment:
@sosnovsky @fcvakintos @ivan-ushakov @Kharchevskyi @ekievsky I have updated the commands to run tests.

So instead of `npm run-script only.test.filter "some test"`
it now is **`npn run-script test.live "some test"`**.